client/config/toml.go

```diff
@@ -0,0 +1,88 @@
+package config
+
+import (
+	"bytes"
+	"os"
+	"path"
+	"text/template"
+
+	"github.com/spf13/viper"
+	tmos "github.com/tendermint/tendermint/libs/os"
+)
+
+const defaultConfigTemplate = `# This is a TOML config file.
+# For more information, see https://github.com/toml-lang/toml
+
+###############################################################################
+###                           Client Configuration                            ###
+###############################################################################
+
+
+chain-id = "{{ .ClientConfig.ChainID }}"
```

Comment:
```suggestion
chain-id = "{{ .ChainID }}"
```

I believe that's the correct template parsing syntax, though not 100% sure

Reply:
I am not sure. I just used tendermint config [approach](https://github.com/cyberbono3/cosmos-sdk/blob/cyberbono3/8529-add-client-config/server/config/toml.go#L21)

Reply:
> I believe that's the correct template parsing syntax, though not 100% sure

Yeah, You suggested the correct approach. Sorry, i was wrong.